bucket/rainbow.json

```
@@ -0,0 +1,14 @@
+{
+    "homepage": "https://github.com/arsham/rainbow",
+    "description": "Tasty rainbows for your terminal!",
```

Do not use this nonsense "marketing" stuff without any real information. User have to know what this application is doing after reading description.

```suggestion
    "description": "Print text in rainbow pattern.",
```

Feel free to come up with valid alternative